content/scripts/check-links.sh

```diff
@@ -3,10 +3,16 @@ set -e
 
 URL=$1
 ACCEPT_PATH=$2
+REJECT_PATH=$3
+
+if [ "$REJECT_PATH" != "" ]; then
```

Comment:
my bash isn't as lit 🔥as yours but I usually did non-empty check like this `if [ -n "$myvar" ]` I assume this works too.